skbio/alignment/_tabular_msa.py

```diff
@@ -318,6 +757,44 @@ def __init__(self, sequences, metadata=None, positional_metadata=None,
         PositionalMetadataMixin._init_(
             self, positional_metadata=positional_metadata)
 
+        # Set up our indexers
+        self._loc = TabularMSALoc(self)
+        self._iloc = TabularMSAILoc(self)
+
+    def _constructor_(self, sequences=NotImplemented, metadata=NotImplemented,
+                      positional_metadata=NotImplemented,
+                      index=NotImplemented):
+        """Return new copy of the MSA with overridden properties.
+
+        NotImplemented is used as a sentinel so that None may be used to
+        override values.
+        """
+        if metadata is NotImplemented:
+            if self.has_metadata():
+                metadata = self.metadata
+            else:
+                metadata = None
+        if positional_metadata is NotImplemented:
+            if self.has_positional_metadata():
+                positional_metadata = self.positional_metadata
+            else:
+                positional_metadata = None
+
+        if index is NotImplemented:
+            if isinstance(sequences, pd.Series):
+                index = sequences.index
+            else:
+                index = self.index
+
+        if sequences is NotImplemented:
+            sequences = self._seqs
+
+        sequences = [copy.copy(s) for s in sequences]
```

Comment:
This should go in `__init__` so that copy behavior is consistent whether you're creating a new `TabularMSA` object or a `TabularMSA` derived from another MSA. Can you add a unit test for `__init__` performing a shallow copy of sequences and mention this behavior in the docs?